### ISSUE TITLE
variation in dS

### ISSUE THREAD

**Issue by ArtPoon** [PoonLab/HexSE#125]
* dS (synonymous substitution rates) is effectively fixed to 1 with no variation
* need to allow user to specify a discretized gamma distribution for dS like we do for dN

**Comment by horaciobam** [comment]
Changes I did in commits b530ae85ec29670dd6f78f7727519df0366326ca and 5762bf2c7a5f70c381622ca4e4d687ed6d3eac3e:

1. Added `dn_values` and `ds_values` to `orf` info when parsing the configuration file. Input file now needs to specify parameters for the distribution from which we will draw dS values:
https://github.com/PoonLab/HexSE/blob/5762bf2c7a5f70c381622ca4e4d687ed6d3eac3e/hexse/settings/__init__.py#L241-L261

2. Assigned `dn` and `ds` values to each codon. I then used them to calculate `omega`:
https://github.com/PoonLab/HexSE/blob/5762bf2c7a5f70c381622ca4e4d687ed6d3eac3e/hexse/sequence_info.py#L608-L624

3. In `get_substitution_rates`, we are not multiplying the `selection_pressure` in a nucleotide by `codon.ds` when mutation is synonymous. We are also storying that value in the `total_omega` variable. 
https://github.com/PoonLab/HexSE/blob/5762bf2c7a5f70c381622ca4e4d687ed6d3eac3e/hexse/sequence_info.py#L371-L389

To do: change names of variables, we are not using `total_omegas` on the event tree, but different combinations of selection pressures.
